registry_module.go

```diff
@@ -269,6 +269,18 @@ type RegistryModuleListOptions struct {
 
 	// Include is a list of relations to include.
 	Include []RegistryModuleListIncludeOpt `url:"include,omitempty"`
+
+	// Search is a search query string. Modules are searchable by name, namespace, provider fields.
+	Search string `url:"q,omitempty"`
+
+	// Provider filters results by provider name
+	Provider string `url:"filter[provider],omitempty"`
+
+	// RegistryName filters results by registry name (public or private)
+	RegistryName string `url:"filter[registry_name],omitempty"`
```

Comment:
I noticed the `RegistryName` field in `RegistryModuleListOptions` uses `string` type instead of the `RegistryName` type used in other areas of the file. This may be intentional for query parameter compatibility, but wanted to confirm in case you weren't aware we had the attribute already!

Reply:
yup, you're correct